qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/IndexingIT.java

```diff
@@ -43,6 +43,8 @@
  * duplication but for now we have no real way to share code.
  */
 public class IndexingIT extends AbstractRollingTestCase {
+
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/59935")
```

Comment:
I think we could remove this after https://github.com/elastic/elasticsearch/pull/59962 is merged, right?

Reply:
Yes, in fact, I was going to unmute it in #59962 itself. I was hoping not to have to mute it here, but I can't get CI to go green since it's failing in most builds of this PR now.